app/assets/javascripts/credit_card_form.js

```diff
@@ -0,0 +1,63 @@
+$(document).on("ready turbolinks:load", function () {
+  let show_error, stripeResponseHandler, submitHandler;
+
+  submitHandler = function (event) {
+    if (!Stripe) {
+      show_error(
+        "Failed to load credit card processing functionality. Please reload this page in your browser."
+      );
+      return;
+    }
+    const $form = $(event.target);
+    $form.find("input[type=submit]").prop("disabled", true);
+
+    //If Stripe was initialized correctly this will create a token using the credit card info
+    Stripe.card.createToken($form, stripeResponseHandler);
+
+    return false;
+  };
+
+  $(".cc_form").on("submit", submitHandler);
+
+  stripeResponseHandler = function (status, response) {
+    let token, $form;
+    $form = $(".cc_form");
```

Comment:
```suggestion
    const $form = $(".cc_form");
```

1. Why `let` for `$form`?
2. You don't need `token` out here. You can set it directly on line 31 since its used there only